kogito-usertasks-with-security-oidc-keycloak/pom.xml

```diff
@@ -0,0 +1,138 @@
+<?xml version="1.0"?>
+<project
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+  xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.kie.kogito.quickstarts</groupId>
+    <artifactId>kogito-quickstarts</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>kogito-usertasks-with-security-oidc-keycloak</artifactId>
+
+  <name>kogito-usertasks-with-security-oidc-keycloak</name>
+  <description>Kogito user tasks orchestration with security enabled on REST api - open id connect adapter(keycloak)</description>
+
+
+  <properties>
+    <quarkus.released.version>0.24.0</quarkus.released.version>
```

Comment:
on master we keep 999-SNAPSHOT in general and then change the version in main pom.xml upon release